src/plugin.js

```diff
@@ -470,64 +493,51 @@ const vr = function(options) {
         window.requestAnimationFrame(animate);
       }
 
-      camera.getWorldDirection(cameraVector);
+      player.vr.camera.getWorldDirection(player.vr.cameraVector);
     }());
+  };
 
-    // Expose the Three.js perspective camera and rotation values on the player under the 'vr' namespace:
-    player.vr = {
-      cameraVector,
-      threeJs: {
-        scene,
-        camera,
-        renderer
-      }
-    };
-
-  }
-
-  player.on('loadstart', function() {
-    initScene();
+  player.on('loadedmetadata', function() {
```

Comment:
we want to re-init the scene every video, but only when we know how big the video will be.